fastestimator/dataset/combined_dataset.py

```diff
@@ -14,70 +14,40 @@
 # ==============================================================================
 
 
-from typing import Any, Dict, Union
+from typing import List
 
-from torch.utils.data import Dataset
+from torch.utils.data import ConcatDataset
 
 from fastestimator.dataset.dataset import FEDataset
 from fastestimator.util.traceability_util import traceable
 
 
 @traceable()
-class CombinedDataset(FEDataset):
-    def __init__(self, datasets: list) -> None:
+class CombinedDataset(ConcatDataset, FEDataset):
```

Comment:
In this case it's no longer an FEDataset. It can just inherit the ConcatDataset. corresponding changes in the unit test text

Reply:
Then we can accept list of dataset instead of list of FEDataset.